test.js

```diff
@@ -9,6 +10,10 @@ describe('sudo mode', function () {
 			return 0;
 		};
 
+		fs.statSync = function () {
```

Comment:
Sorry, this is my first pull request for Node.js code.

I thought it was OK when process.getuid was replaced in all existing tests.

Please give me a hint on what would be an acceptable way to solve this.